src/web/browser/App.tsx

```diff
@@ -42,11 +44,23 @@ export const hydrateApp = ({ CAPI, NAV }: { CAPI: CAPIType; NAV: NavType }) => {
 
 const App = ({ CAPI, NAV }: Props) => {
     const [isSignedIn, setIsSignedIn] = useState<boolean>(false);
+    const [countryCode, setCountryCode] = useState<string>('');
 
     useEffect(() => {
         setIsSignedIn(!!getCookie('GU_U'));
     }, []);
 
+    useEffect(() => {
+        const callFetch = async () => {
+            setCountryCode(await getCountryCode());
+        };
+        callFetch();
+    }, []);
```

Comment:
I think this is a little clearer, but again just my opinion
```
useEffect(async () => {
    try {
        const countryCodeResponse = await getCountryCode()
        setCountryCode(countryCodeResponse)
    } catch (e) {
        // log some error
    }
}, []);
```

Reply:
As per [this great article](https://www.robinwieruch.de/react-hooks-fetch-data), useEffect can't return a promise so the function has to be defined inside